packages/libs/eda/src/lib/core/hooks/workspace.ts

```diff
@@ -154,3 +160,49 @@ function defaultMakeVariableLink({
     ? `/variables/${entityId}`
     : `/variables`;
 }
+
+/**
+ * TODO: This is pasted directly `DefaultVariableRedirect`. Cover this hook by some
+ * kind of test and simplify its logic.
+ */
+export function useGetDefaultVariableDescriptorCallback() {
```

Comment:
While we're moving it. I think we could probably lose the `Callback` at the end of the hook name?

It already has `Get` in the name, suggesting it returns a function.